dev/sparktestsupport/__init__.py

```diff
@@ -19,3 +19,14 @@
 
 SPARK_HOME = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), "../../"))
 USER_HOME = os.environ.get("HOME")
+ERROR_CODES = { "BLOCK_GENERAL" : 10,
```

Comment:
Style nit: the problem with putting the first element of the dict on the same line as the opening `{` is that the indentation of all of the other elements is dependent on the variable name.  If we were to rename `ERROR_CODES` to something else then all of the indentation would have to change.  Instead, I'd move `BLOCK_GENERAL` to the next line and just indent the elements four spaces.